quickstarts/Audio.ipynb

```diff
@@ -17,27 +17,37 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from IPython import display
</pre><p>Maybe nix this import and the display.Markdown below? Just for simplicity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done.</p>